README.md

```diff
@@ -1 +1,46 @@
-# hw01
\ No newline at end of file
+# hw01
+title: "Hw 1"
+author: Zoe Cullison-Shimada  
+  
+# hello my name is?
+I was born and raised in **New York**.
+I am a rising **third-year**.
+I am interested in **languages**. I know _French_ and _Spanish_ but I would love to learn _Japanese_.  
+  
+# something I'd like to learn about? 
+I'd like to take a philsophy class because I've never taken one before. 
+I'd like to investigate my thoughts and carry out a creative inquiry about the world. Thus, a project of mine  
+  
+*The cultural and philsophical undertones of introductions*  
+
+* forms of introduction 
+  + verbal greetings
+  + visual greetings or eye-contact
+  + hug or handshake  
+A footnote [^1]  
+[^1]: Manifestations of introductions from my ethnographic observations  
+
+* cultural undertones
+  + customs and traditions 
+  + conventional behavior
+  + social pressure  
+A footnote [^2]  
+[^2]: My observations are from being in the field. My fieldsite was the UChicago Regenstein Library.  
+
+* philsophical undertones
+  + hospes or hospitality
+  + kindness
+  + friendship  
+A footnote [^3]  
+[^3]: Citations from Cicero  
+
+***  
+# how about an artifact that you're interested in?
```

Comment:
It's best practice to leave a blank line between a header and the next content. This will prevent potential rendering issues